scripts/globals/mobskills/ultimate_terror.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------
+-- Ultimate Terror
+--
+-- Description: AoE Absorb All with randomness
+-- Type: Magical
+---------------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/mobskills")
+---------------------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+
+    local drained = 0
+    local threshold = 3/7
+
+    local effects = {
```

Comment:
The brace on this line needs to be moved down for lua sanity checks

Reply:
Fixed